### PR TITLE
Add setup unit test to make target 'test'

### PR DESCRIPTION
**Pull request by ranakan19**
pre-req: merge -https://github.com/codeready-toolchain/toolchain-e2e/pull/1039 first
wait_test and create_resource_test fixed in https://github.com/codeready-toolchain/toolchain-e2e/pull/1039

### CODE REVIEW

File: make/test.mk [hunk 1]
```diff
@@ -437,5 +437,13 @@ display-eval:
 
 .PHONY: test
 ## Run the unit tests in the 'testsupport/...' packages
-test:
+test: test-support test-setup
+	 @echo 'Run unit tests in test-support and setup'
```
Comment (rajivnathan) [inline make/test.mk:441]:
```suggestion
```
The output actually happens after the tests are executed so might as well remove it

File: setup/metrics/format.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ const MB = 1 << 20
 
 // bytesToMBString converts the given number of bytes to Megabytes and returns it as a string formatted to 2 decimal places
 func bytesToMBString(bytes float64) string {
-	return fmt.Sprintf("%.2f", float64(bytes)/MB)
+	return fmt.Sprintf("%.2f MB", float64(bytes)/MB)
```
Comment (rajivnathan) [inline setup/metrics/format.go:9]: I actually removed the `MB` in https://github.com/codeready-toolchain/toolchain-e2e/commit/12c3a6966a28f9e4bc876833b3b2f1b32687cd8e#diff-ecee3a957ef929c4d71d155152741b178f2566b5cc04cfb0a86ad93b97c50ec3

So let's just remove the `MB` from format_test.go instead.

**Review by rajivnathan — COMMENTED** [review]
Looks good 👍  Just a couple minor comments

File: setup/metrics/format.go (hunk 2 above)
Comment (ranakan19) [reply setup/metrics/format.go:9]: @rajivnathan the unit test fails with the following without MB
```   --- FAIL: TestBytesToMBString/success/zero (0.00s)
            format_test.go:18: 
                        Error Trace:    /Users/krana/go/src/github.com/codeready-toolchain/toolchain-e2e/setup/metrics/format_test.go:18
                        Error:          Not equal: 
                                        expected: "0.00"
                                        actual  : "0.00 MB"
                                        
                                        Diff:
                                        --- Expected
                                        +++ Actual
                                        @@ -1 +1 @@
                                        -0.00
                                        +0.00 MB
                        Test:           TestBytesToMBString/success/zero
```

Comment (ranakan19) [reply setup/metrics/format.go:9]: nvm, i am blind

Comment (ranakan19) [reply setup/metrics/format.go]: done in [ea2ffc7](https://github.com/codeready-toolchain/toolchain-e2e/pull/1038/commits/ea2ffc75f4cc97ce2ad76ebf424647ad8ad67b91)

File: make/test.mk (hunk 1 above)
Comment (ranakan19) [reply make/test.mk]: done in [ea2ffc7](https://github.com/codeready-toolchain/toolchain-e2e/pull/1038/commits/ea2ffc75f4cc97ce2ad76ebf424647ad8ad67b91)

**Review by rajivnathan — APPROVED** [review]
Thanks for catching this! 👍

**Review by mfrancisc — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]
Thanks

**Comment by MatousJobanek** [comment]
/lgtm
/retest 
(infra issue)

**Comment by ranakan19** [comment]
/retest
issue with cluster profiles

**Pull request merged**